FeedStoreChallenge/Helpers/CoreDataHelpers.swift

```diff
@@ -26,3 +26,42 @@ extension NSManagedObjectModel {
 		self.init(contentsOf: momd)
 	}
 }
+
+extension ManagedFeedImage {
+	static func create(with feedImage: LocalFeedImage, in context: NSManagedObjectContext) -> ManagedFeedImage {
+		let image = ManagedFeedImage(context: context)
+		image.id = feedImage.id
+		image.location = feedImage.location
+		image.url = feedImage.url
+		image.desc = feedImage.description
+		return image
+	}
+
+	func toLocalFeedImage() -> LocalFeedImage {
+		return LocalFeedImage(id: id, description: desc, location: location, url: url)
+	}
+}
+
+extension ManagedCache {
+	@discardableResult
+	static func create(with feed: [LocalFeedImage], timestamp: Date, in context: NSManagedObjectContext) -> ManagedCache {
+		let cache = ManagedCache(context: context)
+		let images = feed.map { ManagedFeedImage.create(with: $0, in: context) }
+		cache.images = NSOrderedSet(array: images)
+		cache.timestamp = timestamp
+		return cache
+	}
```

Comment:
Result of this method is not used anywhere thus it is better not to return anything here. If at some moment this requirement appears then it is very easy and fast to change back. 

Please remove returning of a result
```suggestion
	static func create(with feed: [LocalFeedImage], timestamp: Date, in context: NSManagedObjectContext) {
		let cache = ManagedCache(context: context)
		let images = feed.map { ManagedFeedImage.create(with: $0, in: context) }
		cache.images = NSOrderedSet(array: images)
		cache.timestamp = timestamp
	}
```